.travis.yml

```diff
@@ -72,8 +72,17 @@ jobs:
 
     - stage: deploy
       python: '3.6'
-      install: pip install -U setuptools setuptools_scm
+      install: pip install -U setuptools setuptools_scm tox
       script: skip
+      # token to upload github release notes
```

Comment:
Comment should mention the env name (GH_RELEASE_NOTES_TOKEN).

Reply:
Done